awx/ui_next/src/components/Schedule/shared/ScheduleForm.jsx

```diff
@@ -0,0 +1,230 @@
+import React, { useEffect, useCallback } from 'react';
+import { shape, func } from 'prop-types';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { Formik, useField } from 'formik';
+import { Config } from '@contexts/Config';
+import { Form, FormGroup, Title } from '@patternfly/react-core';
+import { SchedulesAPI } from '@api';
+import AnsibleSelect from '@components/AnsibleSelect';
+import ContentError from '@components/ContentError';
+import ContentLoading from '@components/ContentLoading';
+import FormActionGroup from '@components/FormActionGroup/FormActionGroup';
+import FormField, { FormSubmitError } from '@components/FormField';
+import { FormColumnLayout, SubFormLayout } from '@components/FormLayout';
+import { dateToInputDateTime } from '@util/dates';
+import useRequest from '@util/useRequest';
+import { required } from '@util/validators';
+import FrequencyDetailSubform from './FrequencyDetailSubform';
+
+function ScheduleFormFields({ i18n, zoneOptions }) {
+  const [startDateTime, startDateTimeMeta] = useField({
+    name: 'startDateTime',
+    validate: required(
+      i18n._(t`Select a valid date and time for this field`),
+      i18n
+    ),
+  });
+  const [timezone, timezoneMeta] = useField({
+    name: 'timezone',
+    validate: required(i18n._(t`Select a value for this field`), i18n),
+  });
+  const [frequency, frequencyMeta] = useField({
+    name: 'frequency',
+    validate: required(i18n._(t`Select a value for this field`), i18n),
+  });
+
+  return (
+    <>
+      <FormField
+        id="schedule-name"
+        label={i18n._(t`Name`)}
+        name="name"
+        type="text"
+        validate={required(null, i18n)}
+        isRequired
+      />
+      <FormField
+        id="schedule-description"
+        label={i18n._(t`Description`)}
+        name="description"
+        type="text"
+      />
+      <FormGroup
+        fieldId="schedule-start-datetime"
```

Comment:
can we use a datepicker for this

Reply:
:+1: see: https://github.com/ansible/awx/pull/6419#discussion_r398848765

Reply:
Yea, plan is to use this native datepicker for now and pull in PF datepicker when ready